DependencyInjection/Configuration.php

```diff
@@ -388,6 +403,14 @@ private function addExceptionSection(ArrayNodeDefinition $rootNode)
                             ->useAttributeAsKey('name')
                             ->prototype('boolean')->end()
                         ->end()
+                        ->booleanNode('show_exception')
+                            ->treatNullLike($this->debug)
```

Comment:
``$container->getParameter('kernel.debug')`` can be inlined in the xml file.
And the rest can be replaced by
```php
if (null !== $config['exception']['show_exception']) {
   $container->getDefinition('fos_rest.exception.controller')->replaceArgument(2, $config['exception']['show_exception']);
}
```

IMO that's clearer and that way we do not need to add a construtor argument to our ``Configuration`` class.